examples/adaptive_library/DXFImport.py

```diff
@@ -0,0 +1,205 @@
+# custom region from the imported .dxf file.
+#
+# .. image:: ../../images/DXFImport2.png
+#
+# The name of the corresponding .dxf region can be found under **Import** in the Geometry tree in
+# the **Geometry -> Editor -> Geometry** tab in Motor-CAD. Imported regions are displayed by
+# selecting the checkbox. For example, the corresponding imported .dxf region for template region
+# "RotorDuctFluidRegion_2" is named "DXFRegion_Rotor_14".
+#
+# .. image:: ../../images/DXFImport3.png
+#
+# This method will replace the entities (lines and arcs) of "RotorDuctFluidRegion_2" with entities
+# from "DXFRegion_Rotor_14". The geometry will be replaced, while the properties of the region
+# "RotorDuctFluidRegion_2" are retained (name, material, colour etc.).
+
+# %%
+# Get existing template regions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# The regions in the existing template geometry that we wish to replace with custom regions from the
+# imported .dxf geometry will be stored in a list. Use the ``get_region()`` method to get the
+# required regions and store these in the ``standard_regions`` list.
+standard_regions = [
+    mc.get_region("Rotor Pocket_1"),
+    mc.get_region("Rotor Pocket_2"),
+    mc.get_region("Rotor Pocket_4"),
+    mc.get_region("Rotor Pocket_5"),
+    mc.get_region("RotorDuctFluidRegion_2"),
+    mc.get_region("RotorDuctFluidRegion_3"),
+]
+
+# %%
+# Get imported .dxf regions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# The imported .dxf regions that we wish to replace the existing regions with will be stored in the
+# a list. Use the ``get_region_dxf()`` method to get the required regions and store in the
+# ``replacement_regions`` list. The indices of the regions in the ``standard_regions`` and
+# ``replacement_regions`` lists must match such that the corresponding standard and replacement
+# regions share the same index (for example, the "RotorDuctFluidRegion_2" and "DXFRegion_Rotor_14"
+# regions are both the fifth element of their respective lists).
+replacement_regions = [
+    mc.get_region_dxf("DXFRegion_Rotor_6"),
+    mc.get_region_dxf("DXFRegion_Rotor_7"),
+    mc.get_region_dxf("DXFRegion_Rotor_10"),
+    mc.get_region_dxf("DXFRegion_Rotor_11"),
+    mc.get_region_dxf("DXFRegion_Rotor_14"),
+    mc.get_region_dxf("DXFRegion_Rotor_12"),
+]
+
+# %%
+# Replace the ``standard_regions`` with the ``replacement_regions``
+# -----------------------------------------------------------------
+# For each standard region, use the ``Region.replace()`` method to replace the region with its
+# corresponding imported .dxf region. The ``Region.replace()`` method replaces the entities
+# (lines and arcs) of a region with entities from another region. The geometry entities will be
+# replaced, while the properties of the original regions are retained (name, material, colour
+# etc.).
+#
+# For more information on the ``Region.replace()`` method see the entry under ``Region`` in
+# :ref:`ref_geometry_functions` in the :ref:`ref_API_Reference`.
```

Comment:
```suggestion
# For more information on the ``Region.replace()`` method, see the entry under ``Region`` in
# :ref:`ref_geometry_functions` in the API reference documentation.
```
Again, changing only because more than one link is more confusing than helpful.